ui/app/components/secure-variable-form.hbs

```diff
@@ -15,23 +16,42 @@
     </div>
     {{#each this.keyValues as |entry|}}
     <div class="key-value">
+
       <label>
         <span>Key</span>
         <Input
           @type="text"
           @value={{entry.key}}
-          class="input" />
+          class="input"
+          />
       </label>
+
       <label>
-        <span>Value</span>
+        <span>Value
+        </span>
         <Input
-          @type="text"
+          @type={{this.valueFieldType}}
           @value={{entry.value}}
-          class="input" />
+          class="input value-input"
+          autocomplete="new-password" {{!-- prevent auto-fill --}}
+          />
       </label>
+
+      <button
+        class="show-hide-values button is-borderless"
+        type="button"
+        tabindex="-1"
+        {{on "click" this.toggleShowHide}}
+      >
```

Comment:
`thought` `non-blocking`: I noticed that we `each` over every key/value pair in the form. However, this `button` mutates state for all key/value pairs not just the associated `label` and `input` where the click occurs.

Should we convert this `button` into a `toggle` that lives on the top of the form to signify this?

A good example is the dark mode toggle on the top right of this blog:  https://overreacted.io/

Reply:
It's a good thought, and @juliezzhou I defer to your design direction on this one.

My thinking with placing them within the context of the form is they're visually closer to the thing they're showing/hiding, and thus require less explanation. If the eye icon were just at the top of the form alone, I'd be worried that users wouldn't grok what it's for before clicking.